NaReguaApp/src/routes/app-cliente.routes.js

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
```

Comment:
Codacy has a fix for the issue: [Replace `·createNativeStackNavigator·` with `createNativeStackNavigator`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
import {createNativeStackNavigator} from '@react-navigation/native-stack';
```